packages/connect/src/api/signTransaction.ts

```diff
@@ -304,25 +317,14 @@ export default class SignTransaction extends AbstractMethod<'signTransaction', P
             return response;
         }
 
-        let bitcoinTx: Awaited<ReturnType<typeof verifyTx>> | undefined;
+        let bitcoinTx: ReturnType<typeof verifyTx> | undefined;
         if (params.options.decred_staking_ticket) {
-            await verifyTicketTx(
```

Comment:
Are we ok with removing this decred verification?

Reply:
Personally, I am.